helix-term/src/commands/typed.rs

```diff
@@ -1784,6 +1784,7 @@ fn toggle_option(
         .config_events
         .0
         .send(ConfigEvent::Update(config))?;
+    cx.editor.set_status(format!("{} = {}", key, new_value));
```

Comment:
I think this would be clearer as text:

```suggestion
    cx.editor.set_status(format!("Option `{}` is now set to `{}`", key, new_value));
```